java version "1.7.0_67"
Java(TM) SE Runtime Environment (build 1.7.0_67-b01)
Java HotSpot(TM) 64-Bit Server VM (build 24.65-b04, mixed mode)
INFO -  - Configuration path : /home/admin/.areca/workspace/VM_Weekly_Backups/334118795.bcfg
INFO -  - Target : NagiosXI VM
INFO -  - 10%
INFO -  - Backup in progress ...
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
org.apache.commons.net.ftp.FTPConnectionClosedException: FTP response 421 received.  Server closed connection.
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:321)
	at org.apache.commons.net.ftp.FTP._connectAction_(FTP.java:335)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:550)
	at org.apache.commons.net.SocketClient.connect(SocketClient.java:163)
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:267)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:453)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:438)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.list(CachedFileSystemDriver.java:407)
	at com.myJava.file.driver.event.EventFileSystemDriver.list(EventFileSystemDriver.java:430)
	at com.myJava.file.FileSystemManager.list(FileSystemManager.java:363)
	at com.application.areca.impl.AbstractFileSystemMedium.checkRepository(AbstractFileSystemMedium.java:655)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1034)
	at com.application.areca.impl.IncrementalZipMedium.open(IncrementalZipMedium.java:140)
	at com.application.areca.AbstractTarget.open(AbstractTarget.java:326)
	at com.application.areca.impl.FileSystemTarget.open(FileSystemTarget.java:223)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:376)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR
com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:453)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:438)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.list(CachedFileSystemDriver.java:407)
	at com.myJava.file.driver.event.EventFileSystemDriver.list(EventFileSystemDriver.java:430)
	at com.myJava.file.FileSystemManager.list(FileSystemManager.java:363)
	at com.application.areca.impl.AbstractFileSystemMedium.checkRepository(AbstractFileSystemMedium.java:655)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1034)
	at com.application.areca.impl.IncrementalZipMedium.open(IncrementalZipMedium.java:140)
	at com.application.areca.AbstractTarget.open(AbstractTarget.java:326)
	at com.application.areca.impl.FileSystemTarget.open(FileSystemTarget.java:223)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:376)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR
com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:531)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:453)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:438)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.list(CachedFileSystemDriver.java:407)
	at com.myJava.file.driver.event.EventFileSystemDriver.list(EventFileSystemDriver.java:430)
	at com.myJava.file.FileSystemManager.list(FileSystemManager.java:363)
	at com.application.areca.impl.AbstractFileSystemMedium.checkRepository(AbstractFileSystemMedium.java:655)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1034)
	at com.application.areca.impl.IncrementalZipMedium.open(IncrementalZipMedium.java:140)
	at com.application.areca.AbstractTarget.open(AbstractTarget.java:326)
	at com.application.areca.impl.FileSystemTarget.open(FileSystemTarget.java:223)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:376)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	... 14 more
14-10-26 08:42 - ERROR
com.application.areca.ApplicationException: com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1143)
	at com.application.areca.impl.IncrementalZipMedium.open(IncrementalZipMedium.java:140)
	at com.application.areca.AbstractTarget.open(AbstractTarget.java:326)
	at com.application.areca.impl.FileSystemTarget.open(FileSystemTarget.java:223)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:376)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:531)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:453)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:438)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.list(CachedFileSystemDriver.java:407)
	at com.myJava.file.driver.event.EventFileSystemDriver.list(EventFileSystemDriver.java:430)
	at com.myJava.file.FileSystemManager.list(FileSystemManager.java:363)
	at com.application.areca.impl.AbstractFileSystemMedium.checkRepository(AbstractFileSystemMedium.java:655)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1034)
	... 7 more
Caused by: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	... 14 more
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
org.apache.commons.net.ftp.FTPConnectionClosedException: FTP response 421 received.  Server closed connection.
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:321)
	at org.apache.commons.net.ftp.FTP._connectAction_(FTP.java:335)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:550)
	at org.apache.commons.net.SocketClient.connect(SocketClient.java:163)
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:267)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.getRemoteFileInfos(FTPProxy.java:620)
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:160)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.length(FictiveFile.java:117)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.getInformations(AbstractRemoteFileSystemDriver.java:433)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.initEntry(CachedFileSystemDriver.java:680)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.getOrCreateDataEntry(CachedFileSystemDriver.java:652)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:218)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.util.history.HistoryHandler.readHistory(HistoryHandler.java:67)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:98)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.getRemoteFileInfos(FTPProxy.java:620)
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:160)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.length(FictiveFile.java:117)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.getInformations(AbstractRemoteFileSystemDriver.java:433)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.initEntry(CachedFileSystemDriver.java:680)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.getOrCreateDataEntry(CachedFileSystemDriver.java:652)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:218)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.util.history.HistoryHandler.readHistory(HistoryHandler.java:67)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:98)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)INFO -  - Rollbacking backup ...

	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)INFO -  - Rollback completed.

	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - An error occured while reading the target's history
java.lang.IllegalStateException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:170)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.length(FictiveFile.java:117)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.getInformations(AbstractRemoteFileSystemDriver.java:433)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.initEntry(CachedFileSystemDriver.java:680)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.getOrCreateDataEntry(CachedFileSystemDriver.java:652)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:218)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.util.history.HistoryHandler.readHistory(HistoryHandler.java:67)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:98)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
org.apache.commons.net.ftp.FTPConnectionClosedException: FTP response 421 received.  Server closed connection.
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:321)
	at org.apache.commons.net.ftp.FTP._connectAction_(FTP.java:335)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:550)
	at org.apache.commons.net.SocketClient.connect(SocketClient.java:163)
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:267)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.getRemoteFileInfos(FTPProxy.java:620)
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:160)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.exists(FictiveFile.java:137)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.exists(AbstractRemoteFileSystemDriver.java:180)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:224)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.file.FileTool.createDir(FileTool.java:485)
	at com.myJava.util.history.HistoryHandler.writeHistory(HistoryHandler.java:87)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:101)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.getRemoteFileInfos(FTPProxy.java:620)
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:160)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.exists(FictiveFile.java:137)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.exists(AbstractRemoteFileSystemDriver.java:180)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:224)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.file.FileTool.createDir(FileTool.java:485)
	at com.myJava.util.history.HistoryHandler.writeHistory(HistoryHandler.java:87)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:101)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - An error occured while writing the target's history
java.lang.IllegalStateException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractProxy.initFictiveLocalFile(AbstractProxy.java:170)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.initFictiveLocalFile(AbstractRemoteFileSystemDriver.java:196)
	at com.myJava.file.driver.remote.FictiveFile.checkInitialized(FictiveFile.java:132)
	at com.myJava.file.driver.remote.FictiveFile.exists(FictiveFile.java:137)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.exists(AbstractRemoteFileSystemDriver.java:180)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.exists(CachedFileSystemDriver.java:224)
	at com.myJava.file.driver.event.EventFileSystemDriver.exists(EventFileSystemDriver.java:220)
	at com.myJava.file.FileSystemManager.exists(FileSystemManager.java:281)
	at com.myJava.file.FileTool.createDir(FileTool.java:485)
	at com.myJava.util.history.HistoryHandler.writeHistory(HistoryHandler.java:87)
	at com.myJava.util.history.HistoryHandler.addEntryAndFlush(HistoryHandler.java:101)
	at com.application.areca.AbstractTarget.rollbackBackup(AbstractTarget.java:608)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:422)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - AbstractProxy.resetClient() - Client reset because of the following error.
org.apache.commons.net.ftp.FTPConnectionClosedException: FTP response 421 received.  Server closed connection.
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:321)
	at org.apache.commons.net.ftp.FTP._connectAction_(FTP.java:335)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:550)
	at org.apache.commons.net.SocketClient.connect(SocketClient.java:163)
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:267)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:341)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:455)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:430)
	at com.myJava.file.driver.event.EventFileSystemDriver.listFiles(EventFileSystemDriver.java:448)
	at com.myJava.file.FileSystemManager.listFiles(FileSystemManager.java:410)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:832)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:866)
	at com.application.areca.impl.AbstractFileSystemMedium.computeIndicators(AbstractFileSystemMedium.java:280)
	at com.application.areca.AbstractTarget.computeIndicators(AbstractTarget.java:1033)
	at com.application.areca.AbstractTarget.runPostProcessors(AbstractTarget.java:676)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:463)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR
com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:341)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:455)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:430)
	at com.myJava.file.driver.event.EventFileSystemDriver.listFiles(EventFileSystemDriver.java:448)
	at com.myJava.file.FileSystemManager.listFiles(FileSystemManager.java:410)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:832)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:866)
	at com.application.areca.impl.AbstractFileSystemMedium.computeIndicators(AbstractFileSystemMedium.java:280)
	at com.application.areca.AbstractTarget.computeIndicators(AbstractTarget.java:1033)
	at com.application.areca.AbstractTarget.runPostProcessors(AbstractTarget.java:676)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:463)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
14-10-26 08:42 - ERROR - Error while computing statistics.
com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:531)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:341)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:455)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:430)
	at com.myJava.file.driver.event.EventFileSystemDriver.listFiles(EventFileSystemDriver.java:448)
	at com.myJava.file.FileSystemManager.listFiles(FileSystemManager.java:410)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:832)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.listArchives(AbstractIncrementalFileSystemMedium.java:866)
	at com.application.areca.impl.AbstractFileSystemMedium.computeIndicators(AbstractFileSystemMedium.java:280)
	at com.application.areca.AbstractTarget.computeIndicators(AbstractTarget.java:1033)
	at com.application.areca.AbstractTarget.runPostProcessors(AbstractTarget.java:676)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:463)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	... 14 more
INFO -  - Merge in progress ...
INFO -  - Recovery completed - Merged archive creation ...
INFO -  - Committing merge ...
INFO -  - Merge completed
INFO -  - Cleaning old archives ...
INFO -  - Cleaning old archives ...
INFO -  - No archive merged.
INFO -  - Merge completed.
INFO -  - Backup completed.
INFO -  - ------------------------------------------------------------------
INFO -  - An error occurred during the process : com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
INFO -  - Please refer to the log file : /home/admin/.areca/workspace/VM_Weekly_Backups/.log/334118795.14-10-26.log
INFO -  - ------------------------------------------------------------------
14-10-26 08:42 - ERROR
com.application.areca.ApplicationException: com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1143)
	at com.application.areca.impl.IncrementalZipMedium.open(IncrementalZipMedium.java:140)
	at com.application.areca.AbstractTarget.open(AbstractTarget.java:326)
	at com.application.areca.impl.FileSystemTarget.open(FileSystemTarget.java:223)
	at com.application.areca.AbstractTarget.processBackup(AbstractTarget.java:376)
	at com.application.areca.ActionProxy.processBackupOnTarget(ActionProxy.java:98)
	at com.application.areca.launcher.tui.Launcher$1.run(Launcher.java:407)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.myJava.file.driver.remote.UnexpectedConnectionException: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:531)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.lst(CachedFileSystemDriver.java:453)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.listFiles(CachedFileSystemDriver.java:438)
	at com.myJava.file.driver.cache.CachedFileSystemDriver.list(CachedFileSystemDriver.java:407)
	at com.myJava.file.driver.event.EventFileSystemDriver.list(EventFileSystemDriver.java:430)
	at com.myJava.file.FileSystemManager.list(FileSystemManager.java:363)
	at com.application.areca.impl.AbstractFileSystemMedium.checkRepository(AbstractFileSystemMedium.java:655)
	at com.application.areca.impl.AbstractIncrementalFileSystemMedium.open(AbstractIncrementalFileSystemMedium.java:1034)
	... 7 more
Caused by: com.myJava.file.driver.remote.ftp.FTPConnectionException: Error during FTP connection : FTP response 421 received.  Server closed connection.
	at com.myJava.file.driver.remote.ftp.FTPProxy.connect(FTPProxy.java:324)
	at com.myJava.file.driver.remote.ftp.FTPProxy.checkConnection(FTPProxy.java:392)
	at com.myJava.file.driver.remote.ftp.FTPProxy.listFiles(FTPProxy.java:560)
	at com.myJava.file.driver.remote.AbstractRemoteFileSystemDriver.listFiles(AbstractRemoteFileSystemDriver.java:517)
	... 14 more
